client/src/components/Admin/UsersList.jsx

```diff
@@ -0,0 +1,32 @@
+import React, { PropTypes } from 'react';
+import { Link } from 'react-router';
+import { Card, CardActions, CardHeader, CardText } from 'material-ui/Card';
+import FlatButton from 'material-ui/FlatButton';
+
+
+const UsersList = ({ user, deleteUser }) => {
+  const ondeleteUser = () => {
```

Comment:
This method name could be better. From what I can see it's supposed to delete a User so just name it `deleteUser` instead and the callback can simply be `delete`. Also make your methods camelcase i.e this should have been `onDeleteUser`